icons/maneuvers.py

```diff
@@ -1,60 +1,61 @@
-import os
```

Comment:
Github is showing that every line has changed, even ones that haven't. Has something weird happened to the file encoding or something? It makes it pretty hard to review...

Reply:
I've inverted the default emoji color, changing it from a white background (which becomes transparent) to a black background with white text (which is much easier to read on dark mode). Then, I've changed some of the color hex codes to be higher contrast. Additionally, I did a small tweak on the positioning of straight maneuvers (they were too high up on the emoji, so I moved the center down). I also added a glyph anchor for all text to be the middle-middle (anchor="mm") so that everything is properly centered instead of using an arbitrary point as the top-left of the glyph.